src/schema/motor-insurance.json

```diff
@@ -103,6 +103,7 @@
         "Vehicle": {
             "type": "object",
             "description": "Information about the vehicle being insured.",
+            "pii": true,
```

Comment:
In the UK, vehicle licence plates are not PII.

Reply:
Thanks. I'll check with Emma, if we should store it I'll make another change.